kotlin.md

```diff
@@ -156,3 +156,82 @@ fun foo(x: Int, y: String) = Bar().apply{
     this.y = y
 }
 ```
+
+## XML
+
+### 1. Naming `layouts`
+
+Good naming is an absolute must for an Android project. In order to make
+searching for Android layout files easier, we use prefixes corresponding
+to their function. Accordingly:
+
+* `layout_` - Raw layouts; imported via `<layout name="...">`;
+
+* `view_` - View layout files, used by widgets and small self-contained widgets;
+
+* `activity_` - Activity layout files;
+
+* `fragment_` - Fragment layout files;
+
+
+### 2. Naming `drawables`
+
+* `btn_` - Custom button layout;
+
+* `ic_` - Icon layout;
+
+* `bkg_` - Background layout for a view;
+
+* `view_` - Generic view layout (when other type does not fit);
+
+
+### 3. Naming colors
+
+Color names should correspond to color they represent, *not* functionality
+it is related to, such as `appBarColor` or `navBarBackgroundColor`.
+Those are too hard to remember. An example of good naming:
+
+```xml
+<color name="lightCerulean">#7bd3f7</color>
+<color name="cerulean">#019fd9</color>
+<color name="tealBlue">#007bb2</color>
+<color name="pinkishRed">#f31f32</color>
+```
+
+### 4. Naming strings
```

Comment:
Do you plan to add a note about localization? I would be in favour English-first approach (even in applications that are only in Polish). This convention would force the developer to store all strings in resources in English as a base and then creating polish translations.

Reply:
Yes, sure, this branch will be developed further. I want to merge this one though, because I'm switching gears to something else atm.